domain/camera/src/main/java/com/google/jetpackcamera/domain/camera/test/FakeCameraUseCase.kt

```diff
@@ -190,4 +191,8 @@ class FakeCameraUseCase(
             old.copy(captureMode = captureMode)
         }
     }
+
+    override suspend fun setDynamicRange(dynamicRange: DynamicRange) {
+        TODO("Not yet implemented")
```

Comment:
nit: add a bug number to track this todo ?

Reply:
Good catch. I added the implementation.